draft-ietf-capport-architecture.xml

```diff
@@ -268,6 +268,18 @@
         If certificate validation fails, User Equipment MUST NOT make any calls to the API
         server.
         </t>
+        <section>
```

Comment:
Thoughts: does this really deserve its own section in here? Or should I just work it into the section more naturally?